site/jobs/support-engineer.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: job
+title: Support Engineer 
+description: FusionAuth is seeking a Support Engineer to join our team. Learn about this position and apply today.
+department: Engineering
+baseSalary: 60000
+id: 3
+datePosted: 2021-02-02
+dateValidThrough: 2021-12-31
+---
+
+FusionAuth is a fast growing startup headquartered in Denver Colorado. We are expanding our team to help accelerate our growth and take FusionAuth to the next level.
+ 
+Our mission is to make authentication and authorization simple and secure for every developer. This means building the best IAM (Identity and Access Management) and CIAM (Customer Identity and Access Management) solution in the world. If you are passionate about technology and want to join a company that is moving the industry forward, FusionAuth might be a perfect fit for you. 
+
+We are recruiting a Support Engineer who will join our engineering team to help us build the best authentication and authorization solution in the world. This exciting role will allow you to help support our expanding community and help them get the most out of FusionAuth. You will work closely with the software engineers to learn the product and the standard protocols commonly used to integrate with FusionAuth. For this role, you must be a creative and quantitative thinker. You should be interested in the authentication, authorization and user management space. You should be seasoned at writing secure and high-performance code. The ideal candidate will be a self-starter, fast worker and excited about growing a company from the ground up.
+
+You will report directly to the CTO and will work from our Denver office.  
+
+## Responsibilities
+
+Your responsibilities will include:
+
+* Initial response and triage for our various support channels.
+* Identity and correct gaps or typos in our technical documentation.  
+* Producing clear design documentation as needed
+* Work closely with the rest of the engineering team to identify how to best serve our customers
+
+## Opportunities
+
+* Technical writing
+* Squashing bugs
+* Development example applications to demonstrate recommended FusionAuth integration concepts
+
+## Requirements
+
+An ideal candidate will have most of the following qualities or experiences:
+
+* Excellent written and verbal communication
+* Highly motivated
+* A fantastic, customer centric attitude
+* Some programming experience
+* An aptitude for learning new technology
+* An interest in authentication and authorization technology
+
+## Technologies
+
+The requirements for this position include:
+
+* Experience in at least one programming language
+
+## Beneficial Skills
+
+* Technical writing
+* Knowledge of OAuth2 and OpenID Connect
+
+## Beneficial Technologies
+
+* Git
+* Java
+* JavaScript or Node.js
+* HTML
+
+## Salary & Benefits
```

Comment:
Would be awesome to put the interview process in here too.

* how many interviews
* timeframe
* any takehome work?
* etc

Reply:
Added details for take-home and number of interviews. Time frame is hard, no idea when or if we will actually hire someone.  I can add something about timeframe from the start to an answer.